### ISSUE TITLE
[simprod-scripts] Parameter error in GenerateNeutrinos segment (Trac #2187)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2179]
The parameter `RandomService` in the tray segment `GenerateNeutrinos` is not passed to `NuGen`. Thus, the segment only works if a `I3RandomService` is found in the context.

Migrated from https://code.icecube.wisc.edu/ticket/2187
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:23", 
    "description": "The parameter `RandomService` in the tray segment `GenerateNeutrinos` is not passed to `NuGen`. Thus, the segment only works if a `I3RandomService` is found in the context.", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067323910946", 
    "component": "combo simulation", 
    "summary": "[simprod-scripts] Parameter error in GenerateNeutrinos segment", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2018-09-11T09:48:36", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180922T09:51:30: olivas changed milestone from "" to "Winter Solstice 2018"

**Comment by icecube-trac** [comment]
Trac update at 20181126T23:37:19: 
* juancarlos commented:

> In changeset 167097:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="167097"
> Closes #2179. RandomService is now passed to NuGen
> ```


* juancarlos changed status from "new" to "closed"
* juancarlos changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T02:51:31: nwhitehorn commented:

> In changeset 170496:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170496"
> Closes #2179. RandomService is now passed to NuGen
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:23: 
* olivas commented:

> Milestone Winter Solstice 2018 deleted


* olivas changed milestone from "Winter Solstice 2018" to ""
